patterns/base/ul/pattern.yml

```diff
@@ -0,0 +1,11 @@
+name: Unordered List
+description: Unordered list of items
+twig: ./ul.twig
+sass: ./ul.scss
+category: base
+data:
+  ul:
+    class: base--ul
+  li:
+    class: base--li
+    text: List item 1
```

Comment:
@rebmullin you can pass the LI data in using `with` see here:
https://github.com/pattern-library/pattern-library/blob/master/patterns/figure-image/figure-image.twig

`{% set img = figure.img %}` takes the data from the pattern.yml for figure-image

`{% include '../base/img/img.twig' with img %}` then the variable created is used in the included template.